### DISCUSSION TITLE
How to enlarge the icon ?

### DISCUSSION THREAD

**Discussion by kekeimiku** [company-mode/company-mode#1205] [Q&A] [answered]
The icon looks a bit small, I think a bigger one would be nice.

[Imgur](https://i.imgur.com/fh1qrZ8.png)

**Comment by dgutov** [comment, marked as answer]
There is a variable for that. Try:

`M-x customize-variable RET company-icon-size`

To see other customizable variables, `M-x customize-group RET company`.
